common/src/main/java/org/keycloak/common/util/reflections/Types.java

```diff
@@ -593,10 +593,10 @@ private static Type[] extractTypeVariables(Map<String, Type> typeVarMap, Type[]
                 TypeVariable tv = (TypeVariable) types[j];
                 types[j] = typeVarMap.get(tv.getName());
             }
-            else
-            {
-                types[j] = types[j];
-            }
+//            else //useless logic
+//            {
+//                types[j] = types[j];
+//            }
```

Comment:
Missing a description on why this is an issue, and if it is an issue we should ideal have a test-case for it, and remove the line rather than comment it out.